### PR TITLE
DOC-1470: RC Import Data Update

### PR DESCRIPTION
**Pull request by cmilesb**
Staging link: https://docs.redis.com/staging/jira-doc-1470/rc/databases/import-data/

### CODE REVIEW

File: content/rc/databases/import-data.md [hunk 1]
```diff
@@ -29,26 +26,30 @@ As a result, the number of keys in the source and destination databases can be d
 
 To import a dataset from any publicly available Redis server:
 
-1. In the Redis Cloud management console, go to the database that you want to import into.
-1. Click ![Import](/images/rc/icon_import.png#no-click "Import").
+1. Select Databases from the admin console menu and then select the target database from the database list.
```
Comment (rrelledge) [inline content/rc/databases/import-data.md:29]:
```suggestion
1. Select **Databases** from the admin console menu and then select the target database from the database list.
```

File: content/rc/databases/import-data.md [hunk 2]
```diff
@@ -29,26 +26,30 @@ As a result, the number of keys in the source and destination databases can be d
...
-## Import an RDB File
+## Import from a RDB File
 
-You can import any standard RDB to your Redis Cloud instance.
-Both uncompressed and compressed (with a .gz suffix) RDB files are valid.
+If you have a RDB or a compressed RDB file from a Redis database, you can import data from that file into your Redis Enterprise Cloud database.
+
+{{< note >}}
+In order to import a sharded database that has multiple RDB files, you must first merge the files into a single RDB.
+For assistance, contact [Support](https://support.redislabs.com).
```
Comment (rrelledge) [inline content/rc/databases/import-data.md:44]:
```suggestion
For assistance, contact [Support](https://redis.com/company/support/).
```

File: content/rc/databases/import-data.md [hunk 3]
```diff
@@ -61,26 +62,30 @@ To import an RDB file that is stored in an FTP or HTTP server:
...
+    - Select **Add grantee**.
+    - In the Grantee field, enter:
+    `fd1b05415aa5ea3a310265ddb13b156c7c76260dbc87e037a8fc290c3c86b614`
+    - In the Read column, select Yes.
+    - Select Save.
+
+    For more information, see [Configuring ACLs for objects](https://docs.aws.amazon.com/AmazonS3/latest/userguide/managing-acls.html) on the AWS docs.
+
+1. In the [Redis Cloud admin console](https://app.redislabs.com/), select the target database from the database list.
+1. Scroll down to **Danger Zone** and select Import.
```
Comment (rrelledge) [inline content/rc/databases/import-data.md:88]:
```suggestion
1. Scroll down to **Danger Zone** and select **Import**.
```

File: content/rc/databases/import-data.md [hunk 4]
```diff
@@ -91,27 +96,27 @@ To share and import an RDB file that is stored in an AWS Simple Storage Service
...
-1. Go to the GCP console and click on your GCP project.
-1. Click on the menu to open it and select **Storage** to open the Storage browser and view your buckets.
-1. Click on the bucket where the RDB file is stored.
-1. Edit the file permissions:
-    1. Click on the RDB file menu to open it and click **Edit permissions**.
-    1. Click **Add item**.
-    1. Enter the user details and access:
+1. In the GCP Storage bucket, edit the file's Access Control List to give reader access to Redis Enterprise cloud.
+    - Select **Edit access** in the RDB file menu.
+    - Click **Add item**.
```
Comment (rrelledge) [inline content/rc/databases/import-data.md:109]:
```suggestion
    - Select **Add item**.
```

File: content/rc/databases/import-data.md [hunk 5]
```diff
@@ -91,27 +96,27 @@ To share and import an RDB file that is stored in an AWS Simple Storage Service
...
+    - Enter the user details and access:
        - In the Entity field of the new item, select **User**.
        - In the Name field of the new item, enter: `service@redislabs-prod-clusters.iam.gserviceaccount.com`
        - In the Access field of the new item, select **Reader**.
-    1. Click **Save**.
-1. In the Redis Cloud admin management console, select the target database.
-1. Click ![Import](/images/rc/icon_import.png#no-click "Import").
+    - Click **Save**.
```
Comment (rrelledge) [inline content/rc/databases/import-data.md:114]:
```suggestion
    - Select **Save**.
```

File: content/rc/databases/import-data.md [hunk 6]
```diff
@@ -91,27 +96,27 @@ To share and import an RDB file that is stored in an AWS Simple Storage Service
...
+
+    For more information, see [Set ACLs](https://cloud.google.com/storage/docs/access-control/create-manage-lists#set-an-acl) on the Google Cloud docs.
+
+1. In the [Redis Cloud admin console](https://app.redislabs.com/),select the target database from the database list.
+1. Scroll down to **Danger Zone** and select Import.
```
Comment (rrelledge) [inline content/rc/databases/import-data.md:119]:
```suggestion
1. Scroll down to **Danger Zone** and select **Import**.
```

File: content/rc/databases/import-data.md [hunk 7]
```diff
@@ -91,27 +96,27 @@ To share and import an RDB file that is stored in an AWS Simple Storage Service
...
 To share and import an RDB file that is stored in a GCP Storage bucket:
 
-1. Go to the GCP console and click on your GCP project.
-1. Click on the menu to open it and select **Storage** to open the Storage browser and view your buckets.
-1. Click on the bucket where the RDB file is stored.
-1. Edit the file permissions:
-    1. Click on the RDB file menu to open it and click **Edit permissions**.
-    1. Click **Add item**.
-    1. Enter the user details and access:
+1. In the GCP Storage bucket, edit the file's Access Control List to give reader access to Redis Enterprise cloud.
```
Comment (rrelledge) [inline content/rc/databases/import-data.md:107]:
```suggestion
1. In the GCP Storage bucket, edit the file's Access Control List to give reader access to Redis Enterprise Cloud.
```

File: content/rc/databases/import-data.md [hunk 8]
```diff
@@ -91,27 +96,27 @@ To share and import an RDB file that is stored in an AWS Simple Storage Service
...
        - In the Name field of the new item, enter: `service@redislabs-prod-clusters.iam.gserviceaccount.com`
        - In the Access field of the new item, select **Reader**.
-    1. Click **Save**.
-1. In the Redis Cloud admin management console, select the target database.
-1. Click ![Import](/images/rc/icon_import.png#no-click "Import").
+    - Click **Save**.
+
+    For more information, see [Set ACLs](https://cloud.google.com/storage/docs/access-control/create-manage-lists#set-an-acl) on the Google Cloud docs.
+
+1. In the [Redis Cloud admin console](https://app.redislabs.com/),select the target database from the database list.
```
Comment (rrelledge) [inline content/rc/databases/import-data.md:118]:
```suggestion
1. In the [Redis Cloud admin console](https://app.redislabs.com/), select the target database from the database list.
```

**Review by rrelledge — COMMENTED** [review]
Here's a few initial suggestions for a few small things I noticed.

File: content/rc/databases/import-data.md [hunk 9]
```diff
@@ -29,26 +26,30 @@ As a result, the number of keys in the source and destination databases can be d
...
+1. Scroll down to **Danger Zone** and select **Import**.
 1. Enter the source database details:
     - Source Type - Select **Redis**.
     - Redis Hostname/IP Address - Enter the hostname or the public IP address of the source Redis server.
     - Redis port - Enter the port of the source Redis server if it is not the default value of `6379`.
     - Password - Enter the password, if required by the Redis database.
-1. Click **Import**.
+1. Select **Import**.
+
+## Import from an RDB File
```
Comment (rrelledge) [inline content/rc/databases/import-data.md:38]:
```suggestion
## Import from an RDB file
```

File: content/rc/databases/import-data.md [hunk 10]
```diff
@@ -29,26 +26,30 @@ As a result, the number of keys in the source and destination databases can be d
...
-Both uncompressed and compressed (with a .gz suffix) RDB files are valid.
+{{< note >}}
+In order to import a sharded database that has multiple RDB files, you must first merge the files into a single RDB.
+For assistance, contact [Support](https://redis.com/company/support/).
+{{< /note >}}
 
 ### Via FTP or HTTP
 
-To import an RDB file that is stored in an FTP or HTTP server:
+To import an RDB file that is stored on a FTP or HTTP server:
```
Comment (rrelledge) [inline content/rc/databases/import-data.md:49]:
```suggestion
To import an RDB file that is stored on an FTP or HTTP server:
```

File: content/rc/databases/import-data.md [hunk 11]
```diff
@@ -29,26 +26,30 @@ As a result, the number of keys in the source and destination databases can be d
 
 To import a dataset from any publicly available Redis server:
 
-1. In the Redis Cloud management console, go to the database that you want to import into.
-1. Click ![Import](/images/rc/icon_import.png#no-click "Import").
+1. Select **Databases** from the admin console menu and then select the target database from the database list.
+1. Scroll down to **Danger Zone** and select **Import**.
 1. Enter the source database details:
     - Source Type - Select **Redis**.
```
Comment (rrelledge) [inline content/rc/databases/import-data.md:32]:
```suggestion
    - Source type - Select **Redis**.
```

File: content/rc/databases/import-data.md [hunk 12]
```diff
@@ -29,26 +26,30 @@ As a result, the number of keys in the source and destination databases can be d
...
-To import an RDB file that is stored in an FTP or HTTP server:
+To import an RDB file that is stored on a FTP or HTTP server:
 
-1. In the Redis Cloud admin console, select the target database.
-1. Click ![Import](/images/rc/icon_import.png#no-click "Import").
+1. Select **Databases** from the admin console menu and then select your database from the list.
+1. Scroll down to **Danger Zone** and select **Import**.
 1. Enter the details for the RDB file:
     - Source Type - Select **FTP** or **HTTP**.
     - RDB file path - Enter the URL for RDB file: `<protocol>://[username][:password]@[:port]/[path/]filename.rdb[.gz]`
```
Comment (rrelledge) [inline content/rc/databases/import-data.md:55]: It looks like the field name is now `Source path`.
```suggestion
    - Source type - Select **FTP** or **HTTP**.
    - Source path - Enter the URL for the RDB file: `<protocol>://[username][:password]@[:port]/[path/]filename.rdb[.gz]`
```

File: content/rc/databases/import-data.md [hunk 13]
```diff
@@ -61,26 +62,30 @@ To import an RDB file that is stored in an FTP or HTTP server:
...
-1. Click ![Import](/images/rc/icon_import.png#no-click "Import").
+1. In the AWS management console, configure the file’s Access Control List to give read-only access to Redis Enterprise Cloud.
+    - Go to the bucket in the AWS S3 console. In the location where the file is stored, select the RDB file.
+    - Select **Permissions**.
+    - Select **Edit**.
+    - Select **Add grantee**.
+    - In the Grantee field, enter:
+    `fd1b05415aa5ea3a310265ddb13b156c7c76260dbc87e037a8fc290c3c86b614`
+    - In the Read column, select Yes.
+    - Select Save.
```
Comment (rrelledge) [inline content/rc/databases/import-data.md:83]: These should probably still be numbered steps.

File: content/rc/databases/import-data.md [hunk 14]
```diff
@@ -61,26 +62,30 @@ To import an RDB file that is stored in an FTP or HTTP server:
...
+
+    For more information, see [Configuring ACLs for objects](https://docs.aws.amazon.com/AmazonS3/latest/userguide/managing-acls.html) on the AWS docs.
+
+1. In the [Redis Cloud admin console](https://app.redislabs.com/), select the target database from the database list.
+1. Scroll down to **Danger Zone** and select **Import**.
 1. Enter the details for the RDB file:
     - Source Type - Select **AWS S3**.
     - RDB file path - Enter the URL for RDB file: `s3://bucketname/[path/]filename.rdb[.gz]`
```
Comment (rrelledge) [inline content/rc/databases/import-data.md:91]:
```suggestion
    - Source type - Select **AWS S3**.
    - Source path - Enter the URL for the RDB file: `s3://bucketname/[path/]filename.rdb[.gz]`
```

File: content/rc/databases/import-data.md [hunk 15]
```diff
@@ -91,27 +96,27 @@ To share and import an RDB file that is stored in an AWS Simple Storage Service
...
-1. Click on the menu to open it and select **Storage** to open the Storage browser and view your buckets.
-1. Click on the bucket where the RDB file is stored.
-1. Edit the file permissions:
-    1. Click on the RDB file menu to open it and click **Edit permissions**.
-    1. Click **Add item**.
-    1. Enter the user details and access:
+1. In the GCP Storage bucket, edit the file's Access Control List to give read access to Redis Enterprise Cloud.
+    - Select **Edit access** in the RDB file menu.
+    - Select **Add item**.
+    - Enter the user details and access:
```
Comment (rrelledge) [inline content/rc/databases/import-data.md:110]: I would probably change these back to numbered steps.

File: content/rc/databases/import-data.md [hunk 16]
```diff
@@ -91,27 +96,27 @@ To share and import an RDB file that is stored in an AWS Simple Storage Service
...
        - In the Entity field of the new item, select **User**.
        - In the Name field of the new item, enter: `service@redislabs-prod-clusters.iam.gserviceaccount.com`
        - In the Access field of the new item, select **Reader**.
-    1. Click **Save**.
-1. In the Redis Cloud admin management console, select the target database.
-1. Click ![Import](/images/rc/icon_import.png#no-click "Import").
+    - Select **Save**.
```
Comment (rrelledge) [inline content/rc/databases/import-data.md:114]:
```suggestion
    1. Select **Save**.
```

File: content/rc/databases/import-data.md [hunk 17]
```diff
@@ -91,27 +96,27 @@ To share and import an RDB file that is stored in an AWS Simple Storage Service
...
 To share and import an RDB file that is stored in a GCP Storage bucket:
 
-1. Go to the GCP console and click on your GCP project.
-1. Click on the menu to open it and select **Storage** to open the Storage browser and view your buckets.
-1. Click on the bucket where the RDB file is stored.
-1. Edit the file permissions:
-    1. Click on the RDB file menu to open it and click **Edit permissions**.
-    1. Click **Add item**.
-    1. Enter the user details and access:
+1. In the GCP Storage bucket, edit the file's Access Control List to give read access to Redis Enterprise Cloud.
```
Comment (rrelledge) [inline content/rc/databases/import-data.md:107]:
```suggestion
1. In the GCP Storage bucket, edit the file's Access Control List to give read access to Redis Enterprise Cloud:
```

File: content/rc/databases/import-data.md [hunk 18]
```diff
@@ -61,26 +62,30 @@ To import an RDB file that is stored in an FTP or HTTP server:
...
-1. Click on the bucket where the RDB file is stored.
-1. Navigate to the file, select the RDB file and click **Permissions**.
-1. Add access permissions to our service:
-    1. Click **Add account**.
-    1. In the Account field, enter: `fd1b05415aa5ea3a310265ddb13b156c7c76260dbc87e037a8fc290c3c86b614`
-    1. In the Read object column, select **Yes**.
-    1. Click **Save**.
-1. In the Redis Cloud admin console, go to the database that you want to import into.
-1. Click ![Import](/images/rc/icon_import.png#no-click "Import").
+1. In the AWS management console, configure the file’s Access Control List to give read-only access to Redis Enterprise Cloud.
```
Comment (rrelledge) [inline content/rc/databases/import-data.md:75]:
```suggestion
1. In the AWS management console, configure the file’s Access Control List to give read-only access to Redis Enterprise Cloud:
```

File: content/rc/databases/import-data.md [hunk 19]
```diff
@@ -91,27 +96,27 @@ To share and import an RDB file that is stored in an AWS Simple Storage Service
...
-1. Click ![Import](/images/rc/icon_import.png#no-click "Import").
+    - Select **Save**.
+
+    For more information, see [Set ACLs](https://cloud.google.com/storage/docs/access-control/create-manage-lists#set-an-acl) on the Google Cloud docs.
+
+1. In the [Redis Cloud admin console](https://app.redislabs.com/), select the target database from the database list.
+1. Scroll down to **Danger Zone** and select **Import**.
 1. Enter the details for the RDB file:
     - Source Type - Select **Google Cloud Storage**.
     - RDB file path - Enter the URL for RDB file: `gs://bucketname/[path/]filename.rdb[.gz]`
```
Comment (rrelledge) [inline content/rc/databases/import-data.md:122]:
```suggestion
    - Source type - Select **Google Cloud Storage**.
    - Source path - Enter the URL for the RDB file: `gs://bucketname/[path/]filename.rdb[.gz]`
```

File: content/rc/databases/import-data.md [hunk 20]
```diff
@@ -121,14 +126,14 @@ To share and import an RDB file that is stored in a GCP Storage bucket:
...
 
 To import an RDB file that is stored in a Microsoft Azure Blog storage container:
 
-1. In the Redis Cloud admin console, select the target database.
-1. Click ![Import](/images/rc/icon_import.png#no-click "Import").
+1. In the Redis Cloud admin console, select the target database from the database list.
+1. Scroll down to **Danger Zone** and select **Import**.
 1. Enter the details for the RDB file:
     - Source Type - Select **Azure Blob Storage**.
     - RDB file path - Enter the URL for RDB file: `abs://:storage_account_access_key@storage_account_name/[container/]filename.rdb[.gz]`
```
Comment (rrelledge) [inline content/rc/databases/import-data.md:139]:
```suggestion
    - Source type - Select **Azure Blob Storage**.
    - Source path - Enter the URL for the RDB file: `abs://:storage_account_access_key@storage_account_name/[container/]filename.rdb[.gz]`
```

**Review by rrelledge — COMMENTED** [review]
Looking good so far. I left some suggestions for you to consider.

**Review by lanceleonard — APPROVED** [review]

**Pull request merged**